pkg/api/config.go

```diff
@@ -211,6 +211,24 @@ func searchForTestDuplicates(tests []TestStepConfiguration) []error {
 	return nil
 }
 
+func searchForTestConfigDuplicates(tests []TestConfig) []error {
+	duplicates := make(map[string]bool, len(tests))
+	var testNames []string
+
+	for _, test := range tests {
+		if _, exist := duplicates[test.As]; exist {
+			testNames = append(testNames, test.As)
+		} else {
+			duplicates[test.As] = true
+		}
+	}
+
+	if testNames != nil {
+		return []error{fmt.Errorf("tests: found duplicated test: (%s)", strings.Join(testNames, ","))}
+	}
+	return nil
+}
+
 func validateTestConfigurationType(fieldRoot string, test TestStepConfiguration, release *ReleaseTagConfiguration) []error {
```

Comment:
Once we add the `TestFlow` as an option for a `TestStepConfiguration`, we should update the `validateTestConfigurationType` as well